modules/oe_theme_helper/src/TwigExtension/Filters.php

```diff
@@ -59,24 +59,52 @@ public function toNativeLanguageName($language_code): string {
   }
 
   /**
-   * Get file icon given its extension.
+   * Get file icon class given its extension.
    *
    * @param mixed $extension
    *   File extension.
    *
    * @return string
-   *   File icon.
+   *   File icon class name.
    */
-  public function toFileIcon($extension): string {
-    // @todo: Complete and/or centralise extension to icon conversion.
-    // Get icons from given file extension.
-    switch ($extension) {
-      case 'ppt':
-        return 'presentation';
+  public function toFileIcon($extension) {
```

Comment:
Here we are missing `string` type for extension and `: string` return type was removed.